packages/json-data-encoding/json-data-encoding.0.8/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+synopsis: "Type-safe encoding to and decoding from JSON"
+maintainer: "contact@nomadic-labs.com"
+authors: [ "Nomadic Labs" "Ocamlpro" ]
+homepage: "https://gitlab.com/nomadic-labs/json-data-encoding"
+bug-reports: "https://gitlab.com/nomadic-labs/json-data-encoding/issues"
+license: "LGPLv3 w/ linking exception"
+dev-repo: "git+https://gitlab.com/nomadic-labs/json-data-encoding"
+
+build: [
+  ["dune" "build" "-j" jobs "-p" name]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build & >= "1.7"}
```

Comment:
```suggestion
  "dune" {>= "1.7"}
```
Dune should not be tagged as a build dependency (see #14266)

Reply:
Thanks ! Sorry I missed this one when updating all the build files.

I fixed it now.